packages/compass-e2e-tests/positional-args/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+# https://stackoverflow.com/questions/36727740/how-to-hide-console-window-of-a-go-program-on-windows
+GOOS=windows GOARCH=amd64 go build -ldflags -H=windowsgui -o positional-args.exe
```

Comment:
The console window actually still shows but until I added this it would never go away and every time you tried to close it it would open electron again. So it would just leak more and more resources. Now it at least goes away once the program exits and the to be fair the console is actually helpful when debugging because then you can see the output from this program and electron's main process.